packages/assets-controllers/src/TokensController.ts

```diff
@@ -928,7 +929,7 @@ export class TokensController extends BaseController<
             },
           },
         },
-        true,
+        shouldShowRequest,
```

Comment:
This has been referenced in the other PR, but given that ApprovalController parameter is called `shouldShowRequest`, I think we should keep it the same.